### PR TITLE
merge the www to the master

### PR DESCRIPTION
**Pull request by Jack-Wong94**
I have updated the merging branch. Could you check if there is any reference error?

Reference to issue #81

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -27,5 +27,4 @@
 *.mp4
 *.cpuprofile
 node_modules/
-build/webgazer.js
```
Comment (idaknow) [inline .gitignore]: Why was this removed? Shouldn't this be built by the user themselves using grunt.

**Comment by Jack-Wong94** [comment]
I have put the html files back to the www folder

**Comment by idaknow** [comment]
As per my last comment, you still have the build/webgazer.js file pushed to the repository...

File: www/collision.html [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+    #balls {
+      position:absolute;
+      top: 0px;
+      left:0px;
+    }
+  </style>
+</head>
+<body>
+  <script src="js/d3.v3.min.js"></script>
+  <script src="./../build/webgazer.js"></script>
```
Comment (idaknow) [inline www/collision.html]: I feel like we should double check this with Alex in tomorrow's meeting if that path is okay -- but I wouldn't know how else to do it otherwise!
